sycl/doc/extensions/proposed/sycl_ext_oneapi_syclbin.asciidoc

```diff
@@ -0,0 +1,230 @@
+= sycl_ext_oneapi_syclbin
+
+:source-highlighter: coderay
+:coderay-linenums-mode: table
+
+// This section needs to be after the document title.
+:doctype: book
+:toc2:
+:toc: left
+:encoding: utf-8
+:lang: en
+:dpcpp: pass:[DPC++]
+:endnote: &#8212;{nbsp}end{nbsp}note
+
+// Set the default source code type in this document to C++,
+// for syntax highlighting purposes.  This is needed because
+// docbook uses c++ and html5 uses cpp.
+:language: {basebackend@docbook:c++:cpp}
+
+
+== Notice
+
+[%hardbreaks]
+Copyright (C) 2025 Intel Corporation.  All rights reserved.
+
+Khronos(R) is a registered trademark and SYCL(TM) and SPIR(TM) are trademarks
+of The Khronos Group Inc.  OpenCL(TM) is a trademark of Apple Inc. used by
+permission by Khronos.
+
+
+== Contact
+
+To report problems with this extension, please open a new issue at:
+
+https://github.com/intel/llvm/issues
+
+
+== Dependencies
+
+This extension is written against the SYCL 2020 revision 9 specification.  All
+references below to the "core SYCL specification" or to section numbers in the
+SYCL specification refer to that revision.
+
+This extension also depends on the following other SYCL extensions:
+
+* link:../experimental/sycl_ext_oneapi_kernel_compiler.asciidoc[
+  sycl_ext_oneapi_kernel_compiler]
```

Comment:
I don't think there is any hard dependency on the kernel compiler, is there?

Reply:
It's a bit of an odd case, because technically it isn't directly dependent, but the kernel compiler extension is going to be changed to include SYCLBIN file `kernel_bundle` as valid for some of its APIs. You could argue that it means the dependency is the other way around, but it is also something of an interdependence. Maybe we can have a part of this extension that mentions that there are related APIs in the kernel compiler extension instead?

Reply:
Good point. I've removed it from this file.